.gitattributes

```diff
@@ -0,0 +1 @@
+*.pom           text eol=lf
```

Comment:
I will move this file to directory where we need specific line ends.
When we put in on project root - all poms will have lf but other text files in project can be changed by git ...
So project will have mixed line ends.